Gemfile

```diff
@@ -29,7 +29,11 @@ gem 'coffee-rails', '~> 4.1.0'
 # gem 'therubyracer', platforms: :ruby
 
 # use GDS assets, styles etc...
-gem 'govuk_frontend_toolkit', '~> 4.2.1'
+gem 'govuk_frontend_toolkit',
+  git: "https://github.com/alphagov/govuk_frontend_toolkit_gem.git", submodules: true
```

Comment:
I had hell trying to get all the govuk gems and moj_template to actually work. I had to manually hack `Gemfile.lock` in the end.